scripts/linting.sh

```diff
@@ -16,4 +16,10 @@
 ## Shell Opts ----------------------------------------------------------------
 set -euo pipefail
 
+# linting is done in a diferent directory
+sed -i 's/\/opt/\/home\/travis\/build\/rcbops/g' ansible-role-requirements.yml
+
```

Comment:
might it not be cleaner to leave the 'dest' out of the ansible-role-requirements file and use -p when installing with ansible-galaxy?